src/components/DailyHoroscope.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import "../App.css";
+import { PuffLoader } from "react-spinners";
+function DailyHoroscope({ sun_sign, signs, date }) {
+  const [sunSignHoro, setSunSignHoro] = useState([]);
+  const [currentDay, setCurrentDay] = useState("today"); // intializes today as the first state
+  const getHoroscopeData = () => {
+    const urlConst = date.map( // construct url to retrieve data for days in date array
+      (d) => `https://aztro.sameerkumar.website/?sign=${sun_sign}&day=${d}`
+    );
+    Promise.all(
+      urlConst.map((url) =>
+        fetch(url, { method: "POST" }).then((data) => data.json())
+      )
+    )
```

Comment:
I created a fetch for three URL which brings the days "tomorrow","today","yesterday"